tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs

```diff
@@ -14,13 +14,20 @@
 [assembly: OptimizeForBenchmarks]
 [assembly: MeasureInstructionsRetired]
 
-class Tests
+namespace Span
+{
+
+public class Tests
```

Comment:
Can you rename this from Tests to something like SpanBench (or more descriptive and unique). Makes it easier to search, especially in benchview.